warp-tls/Network/Wai/Handler/WarpTLS.hs

```diff
@@ -277,12 +276,8 @@ alpn xs
 ----------------------------------------------------------------
 
 getter :: TLS.TLSParams params => TLSSettings -> Settings -> Socket -> params -> IO (IO (Connection, Transport), SockAddr)
-getter tlsset set sock params = do
-#if WINDOWS
-    (s, sa) <- windowsThreadBlockHack $ accept sock
-#else
-    (s, sa) <- accept sock
-#endif
+getter tlsset set@Settings{settingsAccept = accept'} sock params = do
+    (s, sa) <- accept' sock
```

Comment:
Maybe cleaner? Doesn't matter much, though.

```hs
getter tlsset set sock params = do
    (s, sa) <- settingsAccept set sock
    ...
```

Reply:
Ah, true, it is often used in this module.

Though without it, the diff would be a bit cleaner 😝 but n/m it's fine as it is.